### PR TITLE
Prepare for 1.11.x

### PR DESCRIPTION
**Pull request by stayallive**
You can merge when reviewed 👍

### CODE REVIEW

File: lib/Raven/Client.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 class Raven_Client
 {
-    const VERSION = '1.10.x-dev';
+    const VERSION = '1.11.x-dev';
```
Comment (Jean85) [inline lib/Raven/Client.php]: We forgot to fix this on the tag, right?

Comment (stayallive) [reply lib/Raven/Client.php]: No? https://github.com/getsentry/sentry-php/blob/1.10.0/lib/Raven/Client.php#L19

Comment (Jean85) [reply lib/Raven/Client.php]: Oh you did it without a PR! :+1: Missed it

Comment (stayallive) [reply lib/Raven/Client.php]: Yeah it can be done on the release branch itself so don’t need to PR to do things over there :eyes:

**Pull request merged**